### PR TITLE
Feature/profile card

### PR DESCRIPTION
**Pull request by kodcuserkan**
Sticky profile card has been added to the post page on web view

### CODE REVIEW

**Comment by feruzm** [comment]
always base off new branches from `development` please, not from your lastest PR branches.

File: src/common/pages/entry.tsx [hunk 1]
```diff
@@ -313,6 +357,8 @@ class EntryPage extends BaseComponent<Props, State> {
 
         const {activeUser} = this.props;
 
+        // console.log("authorInfo", authorInfo, "activeUser", activeUser, "state", this);
```
Comment (feruzm) [inline src/common/pages/entry.tsx:360]: please remove this line...

File: src/common/pages/entry.tsx [hunk 2]
```diff
@@ -113,17 +133,41 @@ class EntryPage extends BaseComponent<Props, State> {
...
+        window.removeEventListener("resize", this.detect);
+    }
+
+    detect = () => {
+
+       const infoCard:HTMLElement | null = document.getElementById("avatar-fixed");
+
+       if(infoCard != null && window.scrollY > 180) {
+            infoCard.classList.add('visible')
+       } else if( infoCard != null && window.scrollY <= 180) {
```
Comment (feruzm) [inline src/common/pages/entry.tsx:147]: also make sure we hide info box when reaching comments section. For example, when we reach comments section on this post, http://localhost:3000/hive-196233/@acidyo/another-post-about-beneficiary-usage we shouldn't show info box in comments...

Comment (feruzm) [reply src/common/pages/entry.tsx:147]: Same for viewing comment itself with direct link, should hide when finished/viewing replies of the post... http://localhost:3000/hive-196233/@acidyo/re-singhcapital-qt32wc

Comment (kodcuserkan) [reply src/common/pages/entry.tsx:147]: Ok I see focusing on it

File: src/common/pages/entry.tsx (hunk 1 above)
Comment (kodcuserkan) [reply src/common/pages/entry.tsx]: done

**Comment by feruzm** [comment]
try laptop doesn't show it still. 
image

**Comment by kodcuserkan** [comment]
Ok I am on it

**Comment by kodcuserkan** [comment]
Under 1280px view (until no space then disappears) 

Screen Shot 2021-05-16 at 01 21 17

**Comment by feruzm** [comment]
If there is logged in user (activeUser), are we checking if that user already follows the author or if already favorites user?

**Comment by feruzm** [comment]
Since we don't show author card in mobile view, please make sure we don't do extra request to get author Full account data, so we keep mobile browsing fast without extra requests.

**Comment by kodcuserkan** [comment]
I will check `global.isMobile` later. if that is what we lloking for I will apply

**Review by feruzm — APPROVED** [review]
lgtm, @kodcuserkan I did add `ismobile` checks

**Pull request merged**